straxen/url_config.py

```diff
@@ -428,3 +437,18 @@ def rekey_dict(d, replace_keys='', with_keys=''):
     for old_key, new_key in zip(replace_keys, with_keys):
         new_dict[new_key] = new_dict.pop(old_key)
     return new_dict
+
+
+@URLConfig.register('fixed')
+def get_fixed(name: str):
+    """Return a fixed value for a given name"""
+    from .get_corrections import FIXED_TO_PE
+
+    return FIXED_TO_PE[name]
+
+
+@URLConfig.register('fixed-thresholds')
+def get_thresholds(model: str):
+    """Return a fixed value for a given model"""
+    
+    return straxen.hit_min_amplitude(model)
```

Comment:
perhaps it's worth considering to call this something like `legacy` or something, since we hopefully shouldn't be introducing new ones of this type? I'd prefer less protocols, especially if it's just to keep 1T stuff alive.

Reply:
Yeah good idea. feal free to change it, otherwise will probably have to wait until the weekend when i have some time.

Reply:
Dpone.